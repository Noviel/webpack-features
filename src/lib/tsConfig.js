import path from 'path';

export const tsConfigFile = (type = 'migration') =>
  path.resolve(__dirname, `../../${type}.tsconfig.json`);
