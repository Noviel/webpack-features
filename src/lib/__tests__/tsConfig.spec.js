import { tsConfigFile } from '../tsConfig';

describe('tsConfig', () => {
  it('should return valid path to `migration` config without arguments', () => {
    expect(tsConfigFile()).toMatchSnapshot();
  });

  it('should return valid path to provided config', () => {
    expect(tsConfigFile('strict')).toMatchSnapshot();
  });
});
