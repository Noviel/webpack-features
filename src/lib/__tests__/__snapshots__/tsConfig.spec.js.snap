// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tsConfig should return valid path to \`migration\` config without arguments 1`] = `"C:\\\\projects\\\\dev\\\\webpack-features\\\\src\\\\migration.tsconfig.json"`;

exports[`tsConfig should return valid path to provided config 1`] = `"C:\\\\projects\\\\dev\\\\webpack-features\\\\src\\\\strict.tsconfig.json"`;
